src/components/IconPopup.js

```diff
@@ -0,0 +1,12 @@
+import { h } from 'preact';
+import WdsAvatarIconUser from 'design-system/dist/svg/wds-icons-link.svg';
+
+import './IconPopup.scss';
+
+const IconPopup = ({ position }) => (
+    <div className="pe-icon-popup" style={{ left: position.left, top: position.top }}>
```

Comment:
I would call popup class `pe-popup` so we could reuse it for all (both?) popups.